.test-infra/metrics/sync/github/github_runs_prefetcher/code/main.py

```diff
@@ -404,16 +454,11 @@ def save_workflows(workflows):
                 workflow.url,
                 workflow.category,
                 workflow.threshold,
+                workflow.is_flaky,
             )
         )
-        run_number = 1
-        for run in workflow.runs:
-            if run.id != 0:
```

Comment:
Is it safe to ignore this run.id check? I assume that's filtering out some kind of cancelled or in progress run

Reply:
The idea behind this check is to prevent adding runs with the same run_id, because the implementation used to assign the "0" as the default id. Now this check is no longer applicable because we generate uuid for the id field (https://github.com/apache/beam/pull/30424/files#diff-a99a5e5ac1dc4bf7f4d42360354deec95414d660122bd96214d190b92fee2340R390)